src/main/java/uk/gov/ons/census/casesvc/model/dto/PayloadDTO.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ons.census.casesvc.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import lombok.Data;
+
+@Data
+public class PayloadDTO {
+
+  @JsonInclude(Include.NON_NULL)
```

Comment:
This can't be non-null

Reply:
oh I think I see why you have this annotation... to stop including the null values, sorry

Reply:
this is to stop null being output to event_payload field as per spec